source/Background/services/DashService.ts

```diff
@@ -22,6 +23,8 @@ class DashService {
 
   private _wallet: any;
 
+  private _fiatConversationService = FiatConversionService.getInstance();
```

Comment:
Please use this service independently, there's no need for DashService to know about FiatConversionService